pkg/pillar/types/zedroutertypes.go

```diff
@@ -128,6 +128,53 @@ func (status AppNetworkStatus) VerifyFilename(fileName string) bool {
 			fileName, expect)
 	}
 	return ret
+
+}
+
+// IntfStatusMap - Used to return per-interface errors.
+//  ifName is usually used as the key
+//  In most usecases, Includes entries for all interfaces that were tested.
+//  If an intf is success, ErrorAndTime.Error == "" Else - Set to appropriate Error
+//  ErrorAndTime.ErrorTime will always be set for the interface.
+type IntfStatusMap struct {
+	// StatusMap -> Key: ifname, Value: ErrorAndTime
+	StatusMap map[string]ErrorAndTime
+}
+
+// SetOrUpdateIntfStatus - Add an error into IntfStatusMap. If en entry already
+//  exists for the interface, it is appended.
+// This is for the case of adding errors. To clear error on an interface, use
+//  ClearIntfError.
```

Comment:
There is no ClearIntfError function. What should the comment say?

Reply:
Cleaned up the comment in PR: https://github.com/lf-edge/eve/pull/890